src/account/UpgradeableModularAccount.sol

```diff
@@ -224,9 +257,10 @@ contract UpgradeableModularAccount is
         }
 
         // Run any pre exec hooks for this selector
-        PostExecToRun[] memory postExecHooks = _doPreExecHooks(
+        PostExecToRun[] memory postExecHooks = _doPreHooks(
```

Comment:
Same here, the `executeFromPluginExternal` path should be gone in an earlier PR.